src/lib/x509/certstor.cpp

```diff
@@ -213,4 +214,135 @@ Certificate_Store_In_Memory::Certificate_Store_In_Memory(const std::string& dir)
    }
 #endif
 
+/**
+ * Internal class implementation (i.e. Pimpl) to keep the required platform-
+ * dependent members of Flatfile_Certificate_Store contained in this compilation
+ * unit.
+ */
+class Flatfile_Certificate_Store_Impl
```

Comment:
Technically, I don't really think that we benefit a lot from the PIMPL scheme here. We don't really hide any platform-specifics inside a compilation unit here, anyway.